src/frontend/packages/lib_common/src/cunningham-style.css

```diff
@@ -0,0 +1,39 @@
+/*@import '@openfun/cunningham-react/fonts';*/
```

Comment:
Hi @AntoLC, I wonder why did you comment this? Should it be merged?

Reply:
Hi, the font integration didn't seem to work, we have Roboto on our side as well so I just commented the line to come back on it later on.
I removed it, I will open an issue about it when the PR will be merged.
[fixup! ➕(frontend) add cunningham design system](https://github.com/openfun/marsha/pull/2297/commits/10a013fc30c91ff9f96cc672d01199e252f524e9)